gateway/gateway/schemas.py

```diff
@@ -13,23 +13,23 @@ class CreateOrderSchema(Schema):
     )
 
 
-class GetOrderSchema(Schema):
+class ProductSchema(Schema):
+    id = fields.Str()
+    title = fields.Str()
+    maximum_speed = fields.Int()
+    in_stock = fields.Int()
+    passenger_capacity = fields.Int()
```

Comment:
think we need some `required=True` params here (to match the products-service schema) 
Otherwise, this won't raise any validation errors and the products service does it instead (and messes with our error codes :) )